release-notes/3.0/3.0.0-preview2.md

```diff
@@ -0,0 +1,102 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 3.0.0-preview2.21105.01 released 15 April 2021
+
+This update brings the below changes over the previous release:
+
+### Breaking Changes over Preview V3.0.0-preview1
+- `User Id` now accepts `Client Id` instead of `Object Id` for **User-Assigned Managed Identity**.
+- **SqlDataReader** now returns a `DBNull` value instead of an empty `byte[]`. To active the legacy behaviour, you must enable the following AppContext switch: 
+**"Switch.Microsoft.Data.SqlClient.LegacyRowVersionNullBehaviour"**
+
+### Added
+- Added support for Azure Identity [#1010](https://github.com/dotnet/SqlClient/pull/1010)
+
+### Fixed
+- Fixed wrong data blended with transactions in .NET Core by marking a connection with open result sets to be doomed [#1023](https://github.com/dotnet/SqlClient/pull/1023)
+- Fixed derived parameters containing typename incorrectly [#1020](https://github.com/dotnet/SqlClient/pull/1020)
+- Fixed socket array length to avoid possible out of bounds errors [#1031](https://github.com/dotnet/SqlClient/pull/1031)
+- Fixed **rowversion** null behaviour to return `DBNull` instead of an empty `byte[]` [#998](https://github.com/dotnet/SqlClient/pull/998)
```

Comment:
```suggestion
- Fixed **rowversion** null behavior to return `DBNull` instead of an empty `byte[]` [#998](https://github.com/dotnet/SqlClient/pull/998)
```